web-nfc.d.ts

```diff
@@ -43,25 +43,18 @@ declare interface NDEFRecordInit {
 
 declare type NDEFMessageSource = string | BufferSource | NDEFMessageInit
 
-interface Window {
-  NDEFWriter: NDEFWriter
-}
-declare class NDEFWriter {
-  constructor()
-  write: (
-    message: NDEFMessageSource,
-    options?: NDEFWriteOptions
-  ) => Promise<void>
-}
-
 interface Window {
   NDEFReader: NDEFReader
 }
 declare class NDEFReader extends EventTarget {
   constructor()
-  onerror: (error: any) => void
   onreading: (event: NDEFReadingEvent) => void
+  onreadingerror: (error: any) => void
```

Comment:
`onreadingerror` doesn't expose error or anything actually.

Reply:
should be `onreadingerror: (event: Event) => void` ?